package.json

```diff
@@ -3,23 +3,27 @@
   "version": "0.1.0",
   "license": "Apache 2.0",
   "description": "This JavaScript library provides a simple API to submit and retrieve user data to the Subspace network.",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "main": "./dist/subspace.js",
+  "types": "./dist/subspace.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "rm -rf dist && tsc"
+    "build": "rm -rf dist && tsc && npm run build:browser && npm run build:copy",
+    "build:browser": "browserify --standalone Subspace.js dist/subspace.js -p esmify > dist/subspace.browser.js",
+    "build:copy": "cp dist/subspace.browser.js examples/html-js/src/subspace.browser.js"
   },
   "dependencies": {
     "@polkadot/api": "6.9.2",
-    "@polkadot/ui-keyring": "^0.86.7",
+    "@polkadot/extension-dapp": "^0.41.1",
     "@polkadot/types": "6.9.2",
+    "@polkadot/ui-keyring": "^0.86.7",
     "@polkadot/util": "^7.9.2",
-    "@polkadot/util-crypto": "^7.9.2",
-    "@polkadot/extension-dapp": "^0.41.1"
+    "@polkadot/util-crypto": "^7.9.2"
   },
   "devDependencies": {
+    "browserify": "^17.0.0",
+    "esmify": "^2.1.1",
```

Comment:
I would really prefer webpack usage in the example app instead of pulling browserify here. WDYT?